pkg/app/api/insightstore/filepath_test.go

```diff
@@ -0,0 +1,119 @@
+				dataPointCount: 2,
+				metricsKind:    model.InsightMetricsKind_DEPLOYMENT_FREQUENCY,
+				step:           model.InsightStep_DAILY,
+			},
+			want: []string{"insights/projectID/deployment_frequency/appID/2020-01.json"},
+		},
+		{
+			name: "return correct path with daily and dates that straddles months",
+			args: args{
+				projectID:      "projectID",
+				appID:          "appID",
+				from:           time.Date(2020, 1, 1, 0, 0, 0, 0, time.UTC),
+				dataPointCount: 50,
+				metricsKind:    model.InsightMetricsKind_DEPLOYMENT_FREQUENCY,
+				step:           model.InsightStep_DAILY,
+			},
+			want: []string{"insights/projectID/deployment_frequency/appID/2020-01.json", "insights/projectID/deployment_frequency/appID/2020-02.json"},
+		},
+		{
+			name: "return correct path with weekly",
+			args: args{
+				projectID:      "projectID",
+				appID:          "appID",
+				from:           time.Date(2020, 1, 5, 0, 0, 0, 0, time.UTC),
+				dataPointCount: 2,
+				metricsKind:    model.InsightMetricsKind_DEPLOYMENT_FREQUENCY,
+				step:           model.InsightStep_WEEKLY,
+			},
+			want: []string{"insights/projectID/deployment_frequency/appID/2020-01.json"},
+		},
+		{
+			name: "return correct path with weekly and weeks that straddles months",
+			args: args{
+				projectID:      "projectID",
+				appID:          "appID",
+				from:           time.Date(2020, 1, 5, 0, 0, 0, 0, time.UTC),
+				dataPointCount: 6,
+				metricsKind:    model.InsightMetricsKind_DEPLOYMENT_FREQUENCY,
+				step:           model.InsightStep_WEEKLY,
+			},
+			want: []string{"insights/projectID/deployment_frequency/appID/2020-01.json", "insights/projectID/deployment_frequency/appID/2020-02.json"},
+		},
+		{
+			name: "return correct path with monthly",
+			args: args{
+				projectID:      "projectID",
+				appID:          "appID",
+				from:           time.Date(2020, 1, 1, 0, 0, 0, 0, time.UTC),
+				dataPointCount: 2,
+				metricsKind:    model.InsightMetricsKind_DEPLOYMENT_FREQUENCY,
+				step:           model.InsightStep_MONTHLY,
+			},
+			want: []string{"insights/projectID/deployment_frequency/appID/2020-01.json", "insights/projectID/deployment_frequency/appID/2020-02.json"},
+		},
+		{
+			name: "return correct path with yearly",
+			args: args{
+				projectID:      "projectID",
+				appID:          "appID",
+				from:           time.Date(2020, 1, 1, 0, 0, 0, 0, time.UTC),
+				dataPointCount: 2,
+				metricsKind:    model.InsightMetricsKind_DEPLOYMENT_FREQUENCY,
+				step:           model.InsightStep_YEARLY,
+			},
+			want: []string{"insights/projectID/deployment_frequency/appID/years.json"},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := searchFilePaths(tt.args.projectID, tt.args.appID, tt.args.from, tt.args.dataPointCount, tt.args.metricsKind, tt.args.step); !reflect.DeepEqual(got, tt.want) {
+				t.Errorf("searchFilePaths() = %v, want %v", got, tt.want)
```

Comment:
To unify the output, can you replace it with assert.Equal like:
https://github.com/pipe-cd/pipe/blob/master/pkg/yamlprocessor/yamlprocessor_test.go#L112

Reply:
OK, thanks :D